docs/reference/commandline/run.md

```diff
@@ -83,6 +83,7 @@ parent = "smn_cli"
       --security-opt=[]             Security Options
       --sig-proxy=true              Proxy received signals to the process
       --stop-signal="SIGTERM"       Signal to stop a container
+      --storage-opt=[]              Set storage driver options per container
```

Comment:
Same for this file (description) - can you add the longer description somewhere (possibly in a new section below)